.gitignore

```diff
@@ -6,4 +6,6 @@ venv
 *.pyc
 */bin
 
-notes.notes
\ No newline at end of file
+notes.notes
+# credential json files
+*.json
```

Comment:
Can we change this to `stronk-google-credentials.json` since our README.md tells the user to rename their service account creds file to this?